packages/subgraph/src/dao/utils.ts

```diff
@@ -23,7 +22,9 @@ class WithdrawParams {
   token: Address = Address.fromString(ADDRESS_ZERO);
   to: Address = Address.fromString(ADDRESS_ZERO);
   amount: BigInt = BigInt.zero();
+  /* eslint-disable */
```

Comment:
you can disable a certain rule only for the next line with `// eslint-disable-next-line @typescript-eslint/no-empty-interface`
much cleaner and doesn't have the chance that you can forget to enable it again

Reply:
thanks, amended